internal/dbutil/pgutil/schema.go

```diff
@@ -28,6 +28,15 @@ func ConnstrWithSchema(connstr, schema string) string {
 	return connstr + "&search_path=" + url.QueryEscape(schema)
 }
 
+// ParseSchemaFromConnstr returns the schema name parsed from the connection string
+func ParseSchemaFromConnstr(connstr string) (string, error) {
+	connstrParts := strings.Split(connstr, "&search_path=")
```

Comment:
There's `url.Parse` which allows to properly decode the string. The `search_path` isn't necessarily the last argument.

Reply:
good point